src/ifcwrap/CMakeLists.txt

```diff
@@ -79,7 +79,8 @@ IF(PYTHONINTERP_FOUND AND NOT "${PYTHON_EXECUTABLE}" STREQUAL "")
             OUTPUT_VARIABLE python_package_dir
         )
     ENDIF()
-
+    set(CMAKE_INSTALL_LIBDIR ${CMAKE_INSTALL_PREFIX}/install/lib )
+    set(python_package_dir ${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages/)
```

Comment:
it appears this overrides the `python_package_dir` previously set. This is a bit disruptive to the current build process and workflows. Is there a way we can turn this into a setting.

Also why is CMAKE_INSTALL_PREFIX postfixed with another `/install/` that seems uncommon?

Reply:
yes indeed I need to investigate your cmake. somehow cmake does not install properly the python files and wants to install

Reply:
i turned to change the variable to get rapidly some results

Reply:
> i turned to change the variable to get rapidly some results

I think that's fine, the current approach is to read the dir from the python interpreter, but it is indeed a bit hard to recover probably where it went.

Reply:
@aothms `install` directory is just the name of the directory where to install, below you will see  that the cpack tool actually removes this and set properly the prefix depending on the type of the archive